charms/worker/k8s/src/charm.py

```diff
@@ -133,7 +133,7 @@ def __init__(self, *args):
         self.labeller = LabelMaker(
             self, kubeconfig_path=self._internal_kubeconfig, kubectl=KUBECTL_PATH
         )
-        self._stored.set_default(is_dying=False, cluster_name=str(), annotations={})
```

Comment:
why is this charm stored state?  This doesn't feel right.  I'm not sure why the annotations are stored state either.  Can't we just fetch this from the k8s snap?

Reply:
I adapted here to the existing annotation code but agree - it is weird. Removed it and directly query the API now.